data/src/new_etl/data_utils/recent_activity.py

```diff
@@ -0,0 +1,62 @@
+import pandas as pd
+import requests
+from datetime import datetime, timezone
+
+from ..classes.featurelayer import FeatureLayer
+from ..metadata.metadata_utils import provide_metadata
+from ..constants.services import ACTIVITY_QUERIES
+
+
+def fetch_recent_activity(query: str) -> pd.DataFrame:
+    response = requests.get("https://phl.carto.com/api/v2/sql", params={"q": query})
```

Comment:
We already should have a class that is capable of grabbing a carto query and converting the results to a df. Is there are reason it didn't work for this case?